cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -82,6 +88,19 @@ func (c *Commander) Start(ctx context.Context) error {
 	return c.startNormal()
 }
 
+func (c *Commander) buildConfigs() error {
+	for _, conf := range c.cfg.ConfigFiles {
+		fileName := filepath.Base(conf)
+		newPath := c.logsDir + "/" + fileName
```

Comment:
Can we use filepath.Join
```suggestion
		newPath := filepath.Join(c.logsDir, fileName)
```